pkg/resources/tag_association_acceptance_test.go

```diff
@@ -149,3 +246,131 @@ resource "snowflake_tag_association" "columnTag" {
 }
+
+func tagAssociationConfigColumnIssues1926(tagName, tableName string, databaseName string, schemaName string, columeName string) string {
+	return fmt.Sprintf(`
+resource "snowflake_tag" "test_tag" {
+	name     = "%[1]v"
+	database = "%[3]v"
+	schema   = "%[4]v"
+}
+resource "snowflake_table" "test_table" {
+	name                = "%[2]v"
+	database            = "%[3]v"
+	schema              = "%[4]v"
+	column {
+		name    = "%[5]v"
+		type    = "VARIANT"
+	}
+}
+resource "snowflake_tag_association" "tag_column_association" {
+	object_identifier {
+		database   = "%[3]v"
+		schema     = "%[4]v"
+		name       = "${snowflake_table.test_table.name}.${snowflake_table.test_table.column[0].name}"
+	}
+	object_type = "COLUMN"
+	tag_id      = snowflake_tag.test_tag.id
+	tag_value   = "v1"
+}
+`, tagName, tableName, databaseName, schemaName, columeName)
+}
+
+func tagAssociationConfigColumnIssues1909(tagName, table1Name string, table2Name string, databaseName string, schemaName string, columeName string) string {
+	return fmt.Sprintf(`
+resource "snowflake_tag" "test_tag" {
+	name     = "%[1]v"
+	database = "%[4]v"
+	schema   = "%[5]v"
+}
+resource "snowflake_table" "test_table1" {
+	name                = "%[2]v"
+	database            = "%[4]v"
+	schema              = "%[5]v"
+	column {
+		name    = "%[6]v"
+		type    = "VARIANT"
+	}
+}
+resource "snowflake_table" "test_table2" {
+	name                = "%[3]v"
+	database            = "%[4]v"
+	schema              = "%[5]v"
+	column {
+		name    = "%[6]v"
+		type    = "VARIANT"
+	}
+}
+resource "snowflake_tag_association" "tag_column_association" {
+	object_identifier {
+		database   = "%[4]v"
+		schema     = "%[5]v"
+		name       = "${snowflake_table.test_table1.name}.${snowflake_table.test_table1.column[0].name}"
+	}
+	object_identifier {
+		database   = "%[4]v"
+		schema     = "%[5]v"
+		name       = "${snowflake_table.test_table2.name}.${snowflake_table.test_table2.column[0].name}"
+	}
+	object_type = "COLUMN"
+	tag_id      = snowflake_tag.test_tag.id
+	tag_value   = "v1"
+}
+`, tagName, table1Name, table2Name, databaseName, schemaName, columeName)
+}
+
+func tagAssociationConfigTableIssues1202(tagName, tableName string, databaseName string, schemaName string) string {
+	return fmt.Sprintf(`
+resource "snowflake_tag" "test_tag" {
+	name     = "%[1]v"
+	database = "%[3]v"
+	schema   = "%[4]v"
+}
+resource "snowflake_table" "test_table" {
+	name                = "%[2]v"
+	database            = "%[3]v"
+	schema              = "%[4]v"
+	column {
+		name    = "test_column"
+		type    = "VARIANT"
+	}
+}
+resource "snowflake_tag_association" "tag_column_association" {
+	object_identifier {
+		name = "${snowflake_table.test_table.name}"
+	}
+	object_type = "TABLE"
+	tag_id      = snowflake_tag.test_tag.id
+	tag_value   = "v1"
+}
+`, tagName, tableName, databaseName, schemaName)
+}
+
+func tagAssociationConfigAccountIssues1910(tagName, accountName string, databaseName string, schemaName string) string {
```

Comment:
As far as I remember ORGADMIN role is required for this (?). It's kind of connected to SNOW-1002025, maybe there should be another ticket or some reference created so we don't forget about this.

Reply:
added a ticket